### DISCUSSION TITLE
How to use the multi host connection option?

### DISCUSSION THREAD

**Discussion by TDola** [php-amqplib/php-amqplib#1089] [Q&A]
In the guide it says to use https://github.com/php-amqplib/php-amqplib#multiple-hosts-connections
But that function is deprecated and to use the AMQPConnectionFactory.
How do you use the AMQPConnectionFactory to perform the same task?

Thanks

**Comment by bocall** [comment]
Plus 1 for this. AMQPSSLConnection allowed you to pass multiple hosts but it looks like AMQPConnectionFactory and AMQPConnectionConfig only allow you to pass the one host configuration. Any help in how to use the AMQPConnectionFactory along with AMQPConnectionConfig to connect to multiple hosts would be gratefully appreciated.
